config/crd/bases/druid.gardener.cloud_etcds.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.5.0
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
Just some clarification needed.
Is this an intended change?

Reply:
https://github.com/gardener/etcd-druid/pull/193#discussion_r685698113